lib/spotlight/dor/indexer.rb

```diff
@@ -182,6 +182,52 @@ def add_folder_name(sdb, solr_doc)
       end
     end
 
+    concerning :FullTextIndexing do
+      included do
+        before_index :add_object_full_text
+      end
+
+      # search for configured full text files, and if found, add them to the full text (whole document) solr field
+      def add_object_full_text(sdb, solr_doc)
+        object_level_full_text_urls(sdb).each do |file_url|
+          # append content from each file to the field, creating field if it doesn't exist yet
+          # ruby note: the construct below allows us to append a string to a variable that starts out as nil
+          # because .to_s of nil is empty string
+          solr_doc['full_text_tesim'] = solr_doc['full_text_tesim'].to_s + get_file_content(file_url)
```

Comment:
Is `get_file_content` guaranteed to end with a new line, or is it possible we'll end up with the last word of one file concatenated with the first word of the next?

Also, isn't `_tesim` a multivalued field type? Is there any benefit to storing this as a single value instead of in multiple values? (and, if there is, can we explain that in a comment?)